index.js

```diff
@@ -249,8 +252,11 @@ Plex.prototype.processEvent = function(self, event, sensor) {
         sensor.activePlayers.delete(event.Player.uuid);
         if (sensor.activePlayers.size == 0)
         {
-            sensor.service.getCharacteristic(Characteristic.OccupancyDetected).updateValue(false);
-            self.debugLog("Event triggered sensor off: "+sensor.name);
+            self.debugLog("Event scheduled sensor off: "+sensor.name+" after "+this.delayOff+"ms");
+            this.timeout = setTimeout(function() {
+                self.debugLog("Event triggered sensor off: "+sensor.name);
+                sensor.service.getCharacteristic(Characteristic.OccupancyDetected).updateValue(false);
+            }.bind(this), this.delayOff);
```

Comment:
I don't love that this creates an unnecessary 0 second timeout, and creates duplicated log events, for people with no delay set.

Would you consider moving this functionality around so that a timeout is only set (and the additional `scheduled` log message only shown) when `this.delayOff > 0`?